buildroot-external/board/pc/patches/linux/0001-iwlwifi-Make-missed-beacon-timeout-configurable.patch

```diff
@@ -1,94 +0,0 @@
-From 5712a20822709d1cd744a435eba5d9c8ef91a834 Mon Sep 17 00:00:00 2001
```

Comment:
It sounds like this should have been resolved in 6.10 https://bugzilla.kernel.org/show_bug.cgi?id=203709